locale/it/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: index.hbs
+labels:
+  current-version: Versione attuale
+  download: Download
+  download-for: Download per
+  other-downloads: Altri Downloads
+  changelog: Changelog
+  api: Documentazione API
+---
+
+Node.js® è un JavaScript runtime costruito sul [motore JavaScript V8 di Chrome](https://developers.google.com/v8/).
+Node.js usa un modello senza blocchi I/O e ad eventi, che lo rendono un framework leggero ed efficiente.
```

Comment:
i would go for "Node.js usa un modello di I/O non bloccante e ad event, che lo....."